src/libxtp/davidsonsolver.cc

```diff
@@ -220,7 +365,7 @@ Eigen::MatrixXd DavidsonSolver::QR_ortho(const Eigen::MatrixXd &A) const {
   return result;
 }
 
-Eigen::MatrixXd DavidsonSolver::gramschmidt_ortho(const Eigen::MatrixXd &A,
+Eigen::MatrixXd DavidsonSolver::gs(const Eigen::MatrixXd &A,
```

Comment:
gramschmid is a much better name than gs

Reply:
okidoki